src/Microsoft.Graph.Cli.Core/IO/NativeResponseProcessor.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.Kiota.Abstractions;
+
+namespace Microsoft.Graph.Cli.Core.IO;
+
+public class NativeResponseProcessor : IResponseProcessor
+{
+    public async Task<string?> ExtractStringResponseAsync(IResponseHandler responseHandler)
+    {
+        var handler = responseHandler as NativeResponseHandler;
+        var response = handler?.Value as HttpResponseMessage;
+        if (response is not null)
+        {
+            return await response.Content.ReadAsStringAsync();
+        }
+
+        return null;
+    }
+
+    public async Task<Stream?> ExtractStreamResponseAsync(IResponseHandler responseHandler)
```

Comment:
Since `Stream.Null` is still a stream, I don't think we need to make this nullable.
```suggestion
    public async Task<Stream> ExtractStreamResponseAsync(IResponseHandler responseHandler)
```